sample/TKAccessSamples.m

```diff
@@ -0,0 +1,157 @@
+//
+//  TKAccessSamples.m
+//  TokenSdkTests
+//
+//  Created by Larry Hosken on 11/6/17.
+//  Copyright © 2017 Token Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "TKTestBase.h"
+
+#import "TokenSdk.h"
+#import "TKLogManager.h"
+
+@interface TKAccessSamples : TKTestBase
+
+@end
+
+@implementation TKAccessSamples
+
+-(void)testAccessTokens {
+    TokenIOSync *tokenIOSync = [[self sdkBuilder] buildSync];
+    TKMemberSync *grantorSync = [self createMember:tokenIOSync];
+    // we test getAccounts, so create an account to get:
+    Money *startingBalance = [Money message];
+    startingBalance.currency = @"EUR";
+    startingBalance.value = @"5678.00";
+    [grantorSync linkAccounts: [grantorSync createTestBankAccount:startingBalance]];
+    TKMember *grantor = grantorSync.async;
+    
+    Alias *granteeAlias = [self generateAlias];
+    TKMemberSync *granteeSync = [tokenIOSync createMember:granteeAlias];
+    TKMember *grantee = granteeSync.async;
+  
+    __block Token *accessToken = nil;
+    
+    // createAccessToken begin snippet to include in docs
+    AccessTokenConfig *access = [AccessTokenConfig create:granteeAlias];
+    [access forAllAccounts];
+    
+    [grantor createAccessToken:access
+                     onSuccess:^(Token *t) {
```

Comment:
I notice in some previous diff you used tt as token but here you use t，since you copy these code to the document, do you want to be consistent

Reply:
nice